Adds dagman_progress command line tool

jrbourbeau opened the pull request:
<!-- Thank you for the pull request! -->

#### Reference Issue
<!-- Please provide a link to the respective issue on the issue tracker (https://github.com/jrbourbeau/pycondor/issues) if one exists. For example,

Fixes #<ISSUE_NUMBER>
-->

Fixes #44

#### What does this pull request implement/fix? Explain your changes.

This PR adds a command line tool, `dagman_progress`, that prints a progress bar for submitted `pycondor.Dagman` objects. 

Example use, 
```bash
$ dagman_progress /path/to/dagman/submit_file.submit
[##############################] | 100% complete (14 done, 0 queued, 0 ready, 0 unready, 0 failed)
```

jrbourbeau commented:
@dsschult what do you think?

dsschult commented:
I prefer opening the dagman out file once, instead of multiple times:
https://stackoverflow.com/questions/12523044/how-can-i-tail-a-log-file-in-python/12523119#12523119
Raymond Hettinger is a python dev, so his solution to tailing a file is obviously good.

jrbourbeau commented:
Thanks for the comment @dsschult. I updated `dagman_progress` to only read in the log file once. I also switched to a generator-based method for parsing based on the stackoverflow link you sent.

dsschult commented:
Aren't generators great? :)

jrbourbeau commented:
They're pretty awesome. I'm gonna have to start using them more :tada:

kreczko commented on pycondor/command_line.py:
```diff
@@ -0,0 +1,138 @@
+    return status
+
+
+def status_genreator(dag_out_file):
```
Did you mean `status_generator`?

jrbourbeau replied:
That's definitely what I mean. Good catch @kreczko, thanks!
